pkg/network/ovn_kubernetes_test.go

```diff
@@ -179,6 +179,7 @@ func TestRenderedOVNKubernetesConfig(t *testing.T) {
 		egressIPConfig         *operv1.EgressIPConfig
 		masterIPs              []string
 		v4InternalSubnet       string
+		v4InternalMasqueradeSubnet string
```

Comment:
NIT align all lines above and below this to accommodate the longer string

Reply:
The IDE should do it automagically :)